src/sentry/models/grouplink.py

```diff
@@ -0,0 +1,62 @@
+"""
+sentry.models.groupseen
+~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+from jsonfield import JSONField
+
+from sentry.db.models import Model, sane_repr, BoundedBigIntegerField, BoundedPositiveIntegerField
+
+
+class GroupLink(Model):
+    """
+    Link a group with an extenal resource like a commit, issue, or pull request
+    """
+    __core__ = False
+
+    class Relationship:
+        unknown = 0
+        resolves = 1
+        links = 2
+
+    class LinkedType:
+        unknown = 0
+        commit = 1
+        pull = 2
+        issue = 3
+
+    group_id = BoundedBigIntegerField()
+
+    linked_type = BoundedPositiveIntegerField(
+        default=LinkedType.commit,
+        choices=((LinkedType.commit, _('Commit')),
```

Comment:
i'm pretty sure `choices` are only used for django forms, which we'll probably never have for this, so not sure if this is necessary